### PR TITLE
move to purchase

### PR DESCRIPTION
**Pull request by olenashevchuk**

### CODE REVIEW

**Review by okrdima — APPROVED** [review]

**Review by okrdima — APPROVED** [review]

File: src/app/domains/Cart/components/table/CartTable.template.js [hunk 1]
```diff
@@ -1,20 +1,177 @@
...
+      setStatusMessage({ open: true, message: error, type: 'error' })
+    }
+  }
   if (loading) {
     return <Spinner />
   }
   return (
-    <Table type="cart" products={data} setStatusMessage={setStatusMessage} />
+    <>
+      {data && (
```
Comment (GingerBoy14) [inline src/app/domains/Cart/components/table/CartTable.template.js:150]: You can remove this condition

File: src/app/domains/Wish/components/table/WishTable.template.js [hunk 2]
```diff
@@ -1,27 +1,81 @@
...
+
   return (
     <>
-      <Table
-        type="wishes"
-        products={data}
-        setStatusMessage={setStatusMessage}
-        actions={actions}
-      />
+      {data && (
```
Comment (GingerBoy14) [inline src/app/domains/Wish/components/table/WishTable.template.js:63]: Remove this condition

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

File: src/app/domains/Wish/components/table/WishTable.template.js [hunk 3]
```diff
@@ -1,27 +1,81 @@
...
   )
   if (loading) {
     return <Spinner />
   }
+  const handleMove = async (selectedItems) => {
+    for (let item of selectedItems) {
+      console.log('item', item)
+      try {
+        let product = await getData(COLLECTIONS.WISHES, item)
+        console.log('1, get product')
```
Comment (GingerBoy14) [inline src/app/domains/Wish/components/table/WishTable.template.js:24]: Remove console.log

File: src/app/domains/Wish/components/table/WishTable.template.js [hunk 4]
```diff
@@ -1,27 +1,81 @@
...
+        await setData(COLLECTIONS.CART, item, product)
+        console.log('2, set to cart')
```
Comment (GingerBoy14) [inline src/app/domains/Wish/components/table/WishTable.template.js:26]: Remove console.log

File: src/app/domains/Wish/components/table/WishTable.template.js [hunk 5]
```diff
@@ -1,27 +1,81 @@
...
+        await deleteData(COLLECTIONS.WISHES, item)
+        console.log('3 delete from wish')
```
Comment (GingerBoy14) [inline src/app/domains/Wish/components/table/WishTable.template.js:28]: Remove console.log

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Review by GingerBoy14 — APPROVED** [review]

**Pull request merged**